build/assets/tuned/openshift-node-performance

```diff
@@ -94,8 +94,6 @@ kernel.sched_migration_cost_ns=5000000        # latency-performance
 [selinux]
 avc_cache_threshold=8192                      # Custom (atomic host)
 
-[net]
-nf_conntrack_hashsize=131072                  # Custom (atomic host)
```

Comment:
so this needs to be per-device now, right?

Reply:
yes. and this is manifested in this PR

Reply:
ok, the commit message is a bit too terse on this regard, please make it more explicit. Everything else LGTM, but let me have another pass later.